### PR TITLE
fix cex page for mobile

### LINKED ISSUE #159: [Mobile]: layout instant exchange page
use figma!

this includes the order list page and order details page

Overview page (market overview   recent proposal off) (5)

### LINKED ISSUE #193: querying instant exchange orders triggers the rate limit
we seem to be query the cex api's too frequently

```
2023-10-17 14:06:53.122 [ERR] SHWL: Error getting order info: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-10-17 14:06:53.122 [ERR] SHWL: Error checking for exchange updates: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-10-17 14:07:07.997 [INF] BTC: Current sync progress update is on block 812661, target sync block is 812661
2023-10-17 14:07:13.793 [ERR] SHWL: Error getting order info: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-10-17 14:07:13.793 [ERR] SHWL: Error checking for exchange updates: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-10-17 14:07:34.026 [ERR] SHWL: Error getting order info: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-10-17 14:07:34.026 [ERR] SHWL: Error checking for exchange updates: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-10-17 14:07:49.027 [ERR] SHWL: Error syncing exchange server: failed to sync exchange server [{flypme {false    }}] after 3 attempts
2023-10-17 14:07:49.027 [ERR] UI: Error syncing instant swap: failed to sync exchange server [{flypme {false    }}] after 3 attempts
```

### LINKED ISSUE #285: [Mobile]: populate recent trades section
use figma!

Image

### LINKED ISSUE #314: Screen flickers when clicking on recent order row on Centralized Exchange page
I get something when clicked to recent order row

https://github.com/crypto-power/cryptopower/assets/19331824/48caee0d-3dc7-4de9-8ba6-afa4a86073ce

### PR DESCRIPTION
**Pull request by dreacot**
closes #159 
closes #285
closes #314
closes #193 

Screenshot from 2023-12-13 19-41-54

NB: trade history page and max button functionality would be added via another PR

### CODE REVIEW

**Review by JoeGruffins — COMMENTED** [review]
Unrelated but see this upon navigating to receive. This is the wallets page. Probably fine to ignore:
Screenshot_20231214_152936

**Review by JoeGruffins — APPROVED** [review]

**Comment by dreacot** [comment]
> Unrelated but see this upon navigating to receive. This is the wallets page. Probably fine to ignore:
> Screenshot_20231214_152936

i'm guessing that is a restored wallet, it probably needs to complete syncing first, as address discovery is the last stage during sync for a restored wallet

File: ui/page/root/overview_page.go [hunk 1]
```diff
@@ -943,7 +943,12 @@ func (pg *OverviewPage) recentTrades(gtx C) D {
 
 			return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
 				layout.Rigid(func(gtx C) D {
-					return components.OrderItemWidget(gtx, pg.Load, pg.orders[i])
+					return layout.Inset{
+						Top:    values.MarginPadding6,
+						Bottom: values.MarginPadding6,
+					}.Layout(gtx, func(gtx C) D {
```
Comment (JustinBeBoy) [inline ui/page/root/overview_page.go:949]:
```suggestion
					return components.UniformVeticalInset(values.MarginPadding6).Layout(gtx, func(gtx C) D {
```

File: ui/values/dimensions.go [hunk 2]
```diff
@@ -137,4 +138,52 @@ var (
...
+
+		return size
+	}
+
+	// TextSizeTransform is used to scale text sizes for mobile view.
+	TextSizeTransform = func(isMobileView bool, size unit.Sp) unit.Sp {
+		if isMobileView {
+			switch size {
+			case TextSize12:
+				return TextSize10
```
Comment (JustinBeBoy) [inline ui/values/dimensions.go:163]: I don't think we use size 10 for text on mobile, it too small

**Review by JustinBeBoy — CHANGES_REQUESTED** [review]
1. please check Min max layout and Recent Order, why it can be -1?

image

2. Seem like you missing text size for splash page
b2

**Comment by dreacot** [comment]
> Recent Order, why it can be -1?

that's because of https://github.com/crypto-power/cryptopower/issues/93 @JustinBeBoy

File: ui/page/components/order_list.go [hunk 3]
```diff
@@ -21,12 +21,9 @@ func OrderItemWidget(gtx C, l *load.Load, orderItem *instantswap.Order) D {
 		Height:    cryptomaterial.WrapContent,
 		Alignment: layout.Middle,
 		Border:    cryptomaterial.Border{Radius: cryptomaterial.Radius(14)},
-		Margin: layout.Inset{
-			Top:    values.MarginPadding8,
-			Bottom: values.MarginPadding10,
-		},
 	}.Layout(gtx,
 		layout.Rigid(func(gtx C) D {
```
[the remark this answers is not in the document]
Comment (dreacot) [reply ui/page/components/order_list.go:25]: it wasn't done in the PR, but i'd update it

**Pull request merged**